src/stylesheets/elements/_form.scss

```diff
@@ -232,9 +233,13 @@ button:-moz-focusring,
 .input-group, .input-group__verbose {
   display: flex;
   flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-end;
```

Comment:
radio button, check boxes and selects have now been affect, their styling is off.

Reply:
updated